src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToSubmitHandler.java

```diff
@@ -93,7 +94,7 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
             return response;
         }
 
-        clearExistingBundles(callback);
```

Comment:
There is another PR that removes this function, i.e. removes setting the field to null.
You have not removed that functionality here but **buried** it in  moveExistingBundlesToHistoricalBundles.
It is not clear to the reader what is happening here

Reply:
I think this PR is a replacement for that and 2355 isn't needed